amundsen_application/static/js/ducks/allTags/types.ts

```diff
@@ -1,22 +1,16 @@
-import { Tag } from 'components/Tags/types';
-export { Tag };
+import { Tag } from 'interfaces';
 
-/* API */
-export type AllTagsResponse = {
-  msg: string;
-  tags: Tag[];
-}
-
-/* getAllTags */
 export enum GetAllTags {
-  ACTION = 'amundsen/allTags/GET_ALL_TAGS',
-  SUCCESS = 'amundsen/allTags/GET_ALL_TAGS_SUCCESS',
-  FAILURE = 'amundsen/allTags/GET_ALL_TAGS_FAILURE',
+  REQUEST = 'amundsen/allTags/GET_REQUEST',
```

Comment:
Are we changing `ACTION` -> `REQUEST` across the board or just for HTTP requests?

Reply:
I was going to do it across the board. Any concerns?

Reply:
No concerns here.